File: tkpddiscovery/src/main/java/com/tokopedia/discovery/autocomplete/adapter/decorater/SpacingItemDecoration.java
package com.tokopedia.discovery.autocomplete.adapter.decorater;

import android.graphics.Rect;
import android.support.v7.widget.RecyclerView;
import android.view.View;

public class SpacingItemDecoration extends RecyclerView.ItemDecoration {

    private final int dimen;

    public SpacingItemDecoration(int dimen) {
        this.dimen = dimen;
    }

    public void getItemOffsets(Rect outRect, View view, RecyclerView parent, RecyclerView.State state) {
        outRect.right = this.dimen;
        outRect.top = this.dimen / 2;
        outRect.bottom = this.dimen / 2;
    }
}

File: features/discovery/search/src/main/java/com/tokopedia/search/result/presentation/view/adapter/viewholder/decoration/SpacingItemDecoration.kt
package com.tokopedia.search.result.presentation.view.adapter.viewholder.decoration

import android.graphics.Rect
import android.view.View
import androidx.recyclerview.widget.RecyclerView
import androidx.recyclerview.widget.RecyclerView.ItemDecoration

class SpacingItemDecoration(private val dimen: Int) : ItemDecoration() {

    override fun getItemOffsets(outRect: Rect, view: View, parent: RecyclerView, state: RecyclerView.State) {
        outRect.right = dimen
        outRect.top = dimen / 2
        outRect.bottom = dimen / 2
    }
}